interface/include/log/tfm_log_raw.h

```diff
@@ -35,7 +35,9 @@ extern "C" {
  *                      %c - character
  *                      %% - the '%' symbol
  */
-int tfm_log_printf(const char *fmt, ...);
+int printf(const char *fmt, ...);
+
+#define tfm_log_printf(...) printf(__VA_ARGS__)
```

Comment:
I think that instead of doing such fix we should rather define `NRFX_ASSERT(x)` as `assert(x)`.

Reply:
I prepared #23 with such change.